doc/user/content/overview/materialize-cdc.md

```diff
@@ -0,0 +1,237 @@
+
+The **Materialize CDC format** has been designed to provide a downstream data consumer (like Materialize) with enough information to recognize when records are duplicated or out of order. For a technical deep dive on the subject, see our blog post on [Change Data Capture](https://materialize.com/change-data-capture-part-1/).
+
+Currently, the Materialize CDC format is only supported for [Avro-formatted Kafka sources](/sql/create-source/avro-kafka). If you're interested in using it for another source type, let us know in our [Slack workspace](https://materialize.com/s/chat).
+
+To use the Materialize CDC format, you must:
+
+1. Set up your CDC tool to emit its changefeed in the Materialize CDC format
+2. Define the Materialize CDC format in an Avro schema when you [create a source](../../sql/create-source/avro-kafka/) in Materialize
+
+## Materialize CDC JSON
+
+You use a JSON document to define the format of the record updates transmitted in the changefeed and the format of the separate progress messages which tell the downstream consumer (Materialize) how many updates to expect for a particular logical timestamp. The downstream consumer will then compare the updates it has received to the expected updates and discard duplicates if there are too many.
+
+### Update JSON array
+
+Field | description
+-------- | -----------
+data  | The array that represents the changed record. Each array is composed of objects that define the `name` and `type` of each field to be included in the changefeed.
+time  | The logical timestamp of the update. Materialize uses this to determine the order of updates and, in combination with the progress message, to determine whether there are duplicate updates to be disgarded.
+diff  | The type of update (`-1` for deletion, `1` for addition or upsert)
+
+#### Example update JSON definition
+
+```[
+  {
+    "type": "array",
+    "items": {
+      "type": "record",
+      "name": "update",
+      "namespace": "com.materialize.cdc",
+      "fields": [
+        {
+          "name": "data",
+          "type": {
+            "type": "record",
+            "name": "data",
+            "fields": [
+              {
+                "name": "COLUMNNAME1",
+                "type": "DATATYPE"
+              },
+              {
+                "name": "COLUMNAME2",
+                "type": "DATATYPE"
+              }
+            ]
+          }
+        },
+        {
+          "name": "time",
+          "type": "long"
+        },
+        {
+          "name": "diff",
+          "type": "long"
+        }
+      ]
+    }
+  },
+```
+
+### Progress JSON array
+
+The progress messages describe the number of updates transmitted at a particular timestamp or between specified times.
```

Comment:
I would strike "or" here. The messages describe the number of updates at timestamps between specified times. (and do not describe them outside the specified times).

Reply:
I made this "updates transmitted between timestamps during a specified period" because "timestamps between specified times" looks redundant to me even if it isn't.